### PR TITLE
Gidk 33ftc1 beta

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/assets/fonts/icomoon/fonts/bakup-10march/icomoon.svg [hunk 1]
```diff
@@ -261,6 +261,15 @@
...
 <glyph unicode="&#xe9ee;" glyph-name="company" d="M85.333 362.667c0 70.694 57.308 128 128 128h298.667v-512h-426.667v384zM213.333 405.333c-23.564 0-42.667-19.102-42.667-42.667v-298.667h256v341.333h-213.333zM426.667 789.333c0 70.692 57.306 128 128 128h256c70.694 0 128-57.308 128-128v-810.667h-512v810.667zM554.667 832c-23.565 0-42.667-19.103-42.667-42.667v-725.333h341.333v725.333c0 23.564-19.102 42.667-42.667 42.667h-256zM746.667 277.333h-128v-128h128v128zM746.667 490.667h-128v-128h128v128z" />
+<<<<<<< HEAD
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/assets/fonts/icomoon/fonts/bakup-10march/icomoon.svg:264]: remove HEAD @shalinee-walkover

File: apps/web-giddh/src/assets/fonts/icomoon/fonts/bakup-10march/icomoon.svg [hunk 2]
```diff
@@ -261,6 +261,15 @@
...
+>>>>>>> 4e1667cc59... shalinee | TEST | update all feature popup design and content
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/assets/fonts/icomoon/fonts/bakup-10march/icomoon.svg:272]: remove this @shalinee-walkover

File: apps/web-giddh/src/app/userDetails/components/all-features/all-features.component.html [hunk 3]
```diff
@@ -4,359 +4,396 @@
         <span aria-hidden="true" class="close" data-dismiss="modal" (click)="closePopup()">×</span>
         <h3>{{localeData?.page_heading}}</h3>
     </div>
-    <div id="my-frame" class="modal-body clearfix">
-        <p class="mb-2" [innerHTML]="localeData?.money_back_guarantee"></p>
-
+    <p class="mb-2 money-back-gurantee py-2" [innerHTML]="localeData?.money_back_guarantee"></p>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/userDetails/components/all-features/all-features.component.html:7]:
```suggestion
    <p class="mb-2 money-back-guarantee py-2" [innerHTML]="localeData?.money_back_guarantee"></p>
```

File: apps/web-giddh/src/app/userDetails/components/all-features/all-features.component.scss [hunk 4]
```diff
@@ -1,9 +1,66 @@
...
+            margin-left: 6px;
+        }
+        i.icon-back-arrow{
+            transform: rotate( 180deg);
+            display: none;
+            font-size: 12px;
+            float: left;
+            margin-right: 0;
+        }
+       /* &:hover{
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/userDetails/components/all-features/all-features.component.scss:32]: remove commented code if not required

File: apps/web-giddh/src/assets/css/color.scss [hunk 5]
```diff
@@ -136,3 +136,4 @@ $cornflower: #668cff;
 $grey-chateau: #999da0;
 $persian-blue: #2831b2;
 $link-water: #cbcfda;
+$cascade: #999d9c
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/color.scss:139]:
```suggestion
$cascade: #999d9c;
```

File: apps/web-giddh/src/assets/locale/all-features/hi.json [hunk 6]
```diff
@@ -151,5 +151,24 @@
 	"support": "सहयोग",
 	"talk_to_sales": "सेल्स से बात करें",
 	"help_doc": "सहायता दस्तावेज़",
-	"schedule_demo": "डेमो शेड्यूल करें"
+	"schedule_demo": "डेमो शेड्यूल करें",
+    "purchase_bill": "Purchase Bill",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/all-features/hi.json:155]: Please add hindi translation here

File: apps/web-giddh/src/assets/locale/all-features/mr.json [hunk 7]
```diff
@@ -151,5 +151,24 @@
 	"support": "आधार",
 	"talk_to_sales": "विक्रीशी बोला",
 	"help_doc": "मदत दस्तऐवज",
-	"schedule_demo": "डेमोचे वेळापत्रक तयार करा"
+	"schedule_demo": "डेमोचे वेळापत्रक तयार करा",
+    "purchase_bill": "Purchase Bill",
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/all-features/mr.json:155]: Please add marathi translation here

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/locale/all-features/hi.json (hunk 6 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/locale/all-features/hi.json:155]: Not provided by sandhya, Will update once I get

File: apps/web-giddh/src/assets/locale/all-features/mr.json (hunk 7 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/locale/all-features/mr.json:155]: Not provided by sandhya, Will update once I get

**Review by aditi-tiwari17 — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**